committee.html

```diff
@@ -983,25 +983,20 @@ <h1>THE COMMITTEE</h1>
       </div>
     </div>
 
-    <div class="modal" id="jasel-bio" tabindex="-1" role="dialog"
+    <div class="modal" id="celeste-bio" tabindex="-1" role="dialog"
       aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
       <div class="modal-dialog modal-lg modal-dialog-centered" role="document">
         <div class="modal-content">
           <div class="modal-header">
             <h5 class="modal-title custom-font bold-font darkcolor" style="font-size:160%;"
-              id="exampleModalLongTitle">Jasel Steinmetz</h5>
+              id="exampleModalLongTitle">Celeste Basken</h5>
           </div>
           <div class="modal-body darkcolor medium-font">
-            Hello! My name is Jasel Steinmetz (she/ her) and I am a first-year studying Political
-            Science with a minor in Spanish! I grew up in Southern California in a small beach town
-            called San Clemente, and i’ve been absolutely loving being in the Bay. I am very
-            passionate about social justice and public policy, and outside of ROHP am currently
-            involved with the Cal ALCU on campus, and CUSD Against Racism off campus! On any given
-            day you might find me at Trader Joe’s loading up on snacks, listening to Fleetwood Mac,
-            hiking, cheffing it up in my dorm kitchen, and hanging out with my awesome floor at Unit
-            1. Berkeley is so much fun and I am beyond excited to get to meet and share it with all
-            of you! If you have any questions about anything (classes, majors, RCSA, best food hehe)
-            feel free to reach out! Can’t wait to meet all of you!
+            Celeste is a freshman in Society & Environment, and she's interested in all 
+            things environmental policy and agricultural sustainability! She's from Maryland 
+            (DC area) and will gladly explain the concept of snow and ~weather~ to any 
+            alifornians :) She loves the outdoors, any sort of running around, cooking & 
```

Comment:
there's a typo in her bio - says 'alifornians'. Can you fix this?

Reply:
Yes!